.golangci.yml

```diff
@@ -0,0 +1,22 @@
+run:
```

Comment:
Could you you switch docker image to it as well? Otherwise we’ll have two different configs for two places.

Reply:
I change in Dockerfile